pallets/pips/src/benchmarking.rs

```diff
@@ -232,6 +232,7 @@ benchmarks! {
         let some_url = Some(url.clone());
         let some_desc = Some(description.clone());
         let origin = user.origin();
+        Module::<T>::set_min_proposal_deposit(RawOrigin::Root.into(), 0u32.into()).unwrap();
```

Comment:
Since it's repeated 6x, could you put this in a function?

```suggestion
        set_deposit_to_zero::<T>();
```

Reply:
I've captured this as a new task:
https://polymath.atlassian.net/browse/MESH-1709

For now I am going to merge this PR to make progress w/ the benchmarks.